src/routes/resource-edit.js

```diff
@@ -127,6 +132,7 @@ class ResourceEditRoute extends Component {
         coverageStatement,
         customEmbargoPeriod: customEmbargoPeriod[0] || defaultEmbargoPeriod,
         proxy: { id: proxyId },
+        accessTypeId: accessTypeId === accessTypes.ACCESS_TYPE_NONE_ID ? null : accessTypeId,
```

Comment:
Just suggestion
```suggestion
        accessTypeId: accessTypeId !== accessTypes.ACCESS_TYPE_NONE_ID && accessTypeId,
```

Reply:
It's shorter, but for me if feels a little harder to understand fast